### DISCUSSION TITLE
How to create a Singleton in V?

### DISCUSSION THREAD

**Discussion by Miniputs** [vlang/v#17483] [Questions and Answers] [answered]
I looked at how it is done in Go, but I couldn't get it to work in V. I'm still new to V.
Here is how it can be done in Go: https://stackoverflow.com/questions/1823286/singleton-in-go

**Comment by medvednikov** [comment, marked as answer]
You can do the same with __global and unsafe in V, but it's not recommended.

It's not the V way :)
